### DISCUSSION TITLE
Changing parameters after campaign starts

### DISCUSSION THREAD

**Discussion by idanwekhai** [emdgroup/baybe#334] [General]
I constructed a campaign with all the right parameters and settings and then gave the initial recommendations to the experimentalist. A value was supposed to be continuous and varied for 10 experiments. Still, they just informed me that they kept the value constant for all 10 experiments( They didn't use the recommended values). They've already done the experiments and have objective values ready. Is it possible to account for this change of plan?  I have the saved campaign file in JSON format.

**Comment by Scienfitz** [comment]
Hi @idanwekhai ,

you should be able to utilize these measurements and simply inform the campaign about their outcome via `add_measurements`.

The data you add in that manner **do not have to match** the points that were recommended - in fact you can add measurements for almost arbitrary parameter combinations at any time. They merely have to fit into the parameter values you defined when creating parameters.

**Comment by idanwekhai** [comment]
Thanks for this! I naively assumed that I could only add measurements that were recommended.
